.github/workflows/release.yml

```
@@ -112,7 +112,7 @@ jobs:
         with:
           file: Dockerfile.release
           context: build
-          platforms: linux/amd64,linux/arm64v8
+          platforms: linux/amd64,linux/arm64/v8
```

we can use the alias:
```suggestion
          platforms: linux/amd64,linux/arm64
```